### DISCUSSION TITLE
Tabs Default visible

### DISCUSSION THREAD

**Discussion by flashgordon990** [piroor/treestyletab#3584] [Q&A]
I would like to get TST to show all tabs visible by default and only collapse them if I force that.
Could this be done somehow?

**Comment by irvinm** [comment]
In the TST options ... enable "Unlock Expert Options" and I think you want to uncheck the first 2 items under the "tree Behavior" section.

![image](https://github.com/piroor/treestyletab/assets/979729/dfb25f36-87f8-4c2e-8930-a06bed4bb5c4)
